# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#result definition
int32 total_time
float32[] result_joint_id
float32[] result_joint_angle
